Tests/SilicaTests/Utilities/HTTP/HTTPClient.swift

```diff
@@ -45,7 +46,7 @@ public extension HTTP {
             
             // build request... 
             
-            guard let urlRequest = Foundation.URLRequest(request: request)
+            guard let urlRequest = FoundationNetworking.URLRequest(request: request)
```

Comment:
The module name should be removed. `URLRequest` is enough.

Reply:
It looks like the module name was added because the tests folder has some utility classes that share the same names as URLRequest and URLResponse.

I added a couple of typealias statements to allow the Utilities/HTTP folder URLRequest/Response classes to live alongside the Foundation classes.